### DISCUSSION TITLE
通知一覧から過去のページを見たときに、時々、500Internal Server Errorを返している

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#106] [要望・アイデア・不具合]
通知アイコンをクリックし、「通知一覧を見る」から過去のページを見たときに、時々、500Internal Server Errorを返しているのに気づき、報告させていただきます。以前から観測できていて、再現性のある事象です。

なお、事象が発生しているページは、老番の番号付近のページを指定した際に発生しています。

![スクリーンショット 2022-01-05 23 39 44](https://user-images.githubusercontent.com/38848514/148235987-5528a217-5b58-4a98-8a6c-2d524602c98c.png)

特に困ってはいませんが、お手隙の際にご確認いただければ幸いです。

**Comment by kskwtnk** [comment]
@Brutus03 
ご意見ありがとうございます！

こちら、これまで把握できていなかったので調査いたします 🙇 
またご連絡いたしますのでお待ちいただけますと幸いです！

**Comment by kskwtnk** [comment]
@Brutus03 

お待たせしました！
こちらおそらく原因が分かったのと、修正できそうだったのでIssueを立てて対応いたします！

**Comment by kskwtnk** [comment]
@Brutus03 こちら、修正してリリース完了いたしました！

**Reply by Brutus03** [reply]
こちら、修正されたことを確認しました。迅速な対応ありがとうございます。
